frontend/src/components/uam-configure-group/helpers/get-permission-columns-helper/get-permission-columns.helper.ts

```diff
@@ -23,8 +24,8 @@ const getPermissionsColumns = ({
       Header: 'Select',
       accessor: ({ id }): PermissionsTableActionsProps => ({
         name,
-        onToggle: () => onCheckboxToggle(id),
-        isChecked: selectedPermissionIds.includes(id),
+        onToggle: () => onCheckboxToggle(removeHashtagFromId(id)),
+        isChecked: selectedPermissionIds.includes(removeHashtagFromId(id)),
```

Comment:
Maybe it would be better to have a common `IdCell` component which adds hashtag only on display, instead of changing displayed data type, and then turning it back 🤔